cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -61,6 +61,16 @@ rules:
       - patch
       - update
       - delete
+  - apiGroups:
+      - ''
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
```

Comment:
@nik-kc this may be relevant for https://github.com/kubecost/cost-analyzer-helm-chart/pull/2235